src/test/regress/expected/multi_insert_select.out

```diff
@@ -684,15 +684,12 @@ INSERT INTO agg_events
 DEBUG:  CTE sub_cte is going to be inlined via distributed planning
 DEBUG:  Subqueries without relations are not allowed in distributed INSERT ... SELECT queries
 DEBUG:  Router planner cannot handle multi-shard select queries
-DEBUG:  Subqueries without relations are not allowed in distributed INSERT ... SELECT queries
-DEBUG:  CTE sub_cte is going to be inlined via distributed planning
-DEBUG:  Router planner cannot handle multi-shard select queries
-DEBUG:  Router planner cannot handle multi-shard select queries
-DEBUG:  generating subplan XXX_1 for CTE sub_cte: SELECT 1
-DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT user_id, (SELECT sub_cte."?column?" FROM (SELECT intermediate_result."?column?" FROM read_intermediate_result('XXX_1'::text, 'binary'::citus_copy_format) intermediate_result("?column?" integer)) sub_cte("?column?")) AS value_1_agg FROM public.raw_events_first
-DEBUG:  Router planner cannot handle multi-shard select queries
-ERROR:  could not run distributed query with subquery outside the FROM, WHERE and HAVING clauses
-HINT:  Consider using an equality filter on the distributed table's partition column.
+DEBUG:  performing repartitioned INSERT ... SELECT
```

Comment:
I guess we skip INSERT .. SELECT pushdown with a minor issue?

```SQL
insert into users_table(user_id) SELECT  (SELECT e.user_id)
FROM events_table e;

...
DEBUG:  Subqueries without relations are not allowed in distributed INSERT ... SELECT queries

```

Is that something that can we do easily?

Reply:
Would prefer to do it in a separate PR. I think we need to do some general consolidation between insert..select and subquery pushdown.

Reply:
Makes sense, marking https://github.com/citusdata/citus/issues/1428 as relevant